packages/grafana-llm-app/src/components/AppConfig/HealthCheck.tsx

```diff
@@ -128,6 +137,8 @@ function ShowOpenAIHealth({ openAI }: { openAI: OpenAIHealthDetails | boolean })
           </li>
         ))}
       </div>
+      <b>Assistant: </b>
+      {openAI.assistant.ok ? 'OK' : `Error: ${openAI.assistant.error}`}
```

Comment:
```suggestion
      {openAI.assistant.ok ? 'OK' : `Error: ${openAI.assistant.error}. The configured OpenAI provider may not offer assistants APIs.`}
```

Reply:
Agree, this is a more precise description.